### DISCUSSION TITLE
Difficulty in get the price of ETH in USD

### DISCUSSION THREAD

**Discussion by azeezz0090** [Cyfrin/foundry-full-course-cu#2046] [Q&A] [answered]
Please help me get the price of ETH in USD; it seems like there is a change on ChainLink

**Comment by EngrPips** [comment]
Hello @azeezz0090, what change did you notice on ChainLink?

**Reply by azeezz0090** [reply]
Hi @EngrPips, thanks for the prompt response.
The code to for getting the price of ETH in USD is now for getting BTC in USD.

**Reply by EngrPips** [reply, marked as answer]
Are you sure you are not using the priceFeed address of `BTC/USD` for `ETH/USD.` Please check that you are using the correct price feed address.

**Reply by azeezz0090** [reply]
@EngrPips thank you very much.

**Reply by EngrPips** [reply]
Awesome, You are welcome, Friend.
